opendevin/runtime/client/client.py

```diff
@@ -61,55 +62,102 @@ def __init__(self, plugins_to_load: list[Plugin], work_dir: str) -> None:
 
     def _init_bash_shell(self, work_dir: str) -> None:
         self.shell = pexpect.spawn('/bin/bash', encoding='utf-8', echo=False)
-        self.__bash_PS1 = r'[PEXPECT_BEGIN] \u@\h:\w [PEXPECT_END]'
```

Comment:
@xingyaoww why set the prompt up so complex to begin with? 🤔 
Doesn't this make detection harder than it should be?

Reply:
> just to avoid PS1= ... :D
It is just the agent could possibly run `env` to check the env var, then found itself dead in the water because of the PS1 pattern 😓 

> why set the prompt up so complex to begin with? 

Do you mean the `user@host:/workdir` thing, or the `[PEXPECT_BEGIN]`? For the former, i think it will be useful for the agent to know whoami and where am i - so i keep those user/host/workdir info. For `[PEXPECT_BEGIN]` - i spend sometime and i think that's probably the easiest way to make the `\u@\h:\w` regex work.

Reply:
Yes, user@host. But I see the advantage in extra information for the agent there, which seems the obvious way to do now that you mention it! 👍